packages/server/src/games/quiz.ts

```diff
@@ -158,10 +160,23 @@ export default class Quiz extends Room {
   playerGuess = (id: string, guess: string) => {
     const player = this.getPlayer(id);
     if (!guess || !player || this.answers.length < 1) return;
+    if (
+      this.isGuessTime === false ||
+      player.canGuess === false ||
+      player.numberOfGuess > GAME_MAX_GUESS
+    )
+      return;
     const result = stringSimilarity.findBestMatch(guess.toLowerCase(), this.answers);
-    if (player.canGuess === true && result.bestMatch.rating >= 0.8 && this.isGuessTime === true) {
+
+    if (result.bestMatch.rating === 1) {
+      // correct answer
       this.currentNumberOfValidAnswers++;
       this.playerFoundAnswer(player, this.currentNumberOfValidAnswers);
+    } else if (result.bestMatch.rating >= 0.8) {
+      // almost correct answer
+    } else {
+      // bad answer
+      player.numberOfGuess++;
```

Comment:
nit: Do we want to keep this variable as private and maybe use a method `failedGuess` that will increase this quantity? This is not an issue just a suggestion :D

Reply:
I like the idea.

What about creating also an other function to tell if a player can guess (function name ?) ?
```
fn canGuess (maxGuess): bool ->
    return this.canGuess && this.numberOfGuess <= maxGuess
```

Reply:
Sounds good to me too